CHEWBBACA/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "2.0.17.2"
+__version__ = "2.1.0"
```

Comment:
:+1:  Good to be in agreement with (semantic versioning guidelines)[https://semver.org/]